packages/lib/CalEventParser.ts

```diff
@@ -205,6 +204,12 @@ ${calEvent.paymentInfo.link}
     : ""
 }
   `.trim();
+
+  const additionalNotes = getAdditionalNotes(calEvent, t);
```

Comment:
We shouldn't change the order of the sections.

Reply:
Got it, I think we can convert the return value for getAdditionalNotes to a boolean and render it conditionally in that order.

Reply:
fixed in commit #[1f6b642](https://github.com/calcom/cal.com/pull/9237/commits/1f6b642afa6a5802dde8555711c6fae28255d9e5)

Reply:
@hariombalhara fixed in commit #[1f6b642] (https://github.com/calcom/cal.com/pull/9237/commits/1f6b642afa6a5802dde8555711c6fae28255d9e5)